src/tests/unit/tests/electron/platform/android/scan-results-helpers.ts

```diff
@@ -129,12 +128,12 @@ export function buildRuleInformation(
             const summary: string = 'How to fix ' + ruleId;
             return ({ howtoFixSummary: summary } as unknown) as UnifiedResolution;
         },
-        includeThisResultDelegate: r => {
+        getResultStatusDelegate: r => {
             expect('includeThisResultDelegate').toBe('This code should never execute');
             return null;
         },
-        includeThisResult: r => {
-            return includeResults;
+        getResultStatus: r => {
+            return r.status;
```

Comment:
This created snapshots which made me concerned there was an implementation bug, since r.status isn't the expected return type of getResultStatus; can this maybe do something like `r.status === 'PASS' ? 'pass' : 'unknown'` instead?

Reply:
Updated! I used a slightly different conditional that is closer to what happens in prod code